docs/tomcat.md

```diff
@@ -57,15 +57,15 @@ To configure a receiver for your tomcat system logs, specify the following field
 | Field                 | Default                           | Description |
 | ---                   | ---                               | ---         |
 | `type`                | required                          | Must be `tomcat_system`. |
-| `include_paths`       | `[/opt/tomcat/logs/catalina.out]` | A list of filesystem paths to read by tailing each file. A wild card (`*`) can be used in the paths; for example, `/var/log/apache*/*.log`.
+| `include_paths`       | `[/var/log/tomcat9/catalina.out]` | A list of filesystem paths to read by tailing each file. A wild card (`*`) can be used in the paths; for example, `/var/log/apache*/*.log`.
```

Comment:
We changed this to wildcard in #342, right?

Reply:
yes, looks like the doc update was missed